tests/test_bot.py

```diff
@@ -363,6 +364,56 @@ def test_send_reply(self):
                                 attachments='null')
 
 
+class TestSendMessageWithMessageAttachments(TestPhialBot):
+    '''Test phial's send_message function with message attachments'''
+
+    def test_send_message(self):
+        self.bot.slack_client = MagicMock()
+        self.bot.slack_client.api_call = MagicMock(return_value="test")
+        message = Response(channel="channel_id",
+                           attachments=[MessageAttachment(
+                                fallback="fallback",
+                                author_name="John Doe",
+                                author_link="https://example.com/author",
+                                author_icon="https://example.com/author.jpg",
+                                color="#36a64f",
+                                title="Title",
+                                title_link="https://example.com",
+                                image_url="https://example.com/image.jpg",
+                                text="Go to Example Website",
+                                footer="Footer text",
+                                footer_icon="https://example.com/footer.jpg",
+                                thumb_url="https://example.com/thumb.jpg",
+                                fields=[
+                                    MessageAttachmentField(
+                                        title="Established",
+                                        value="2008",
+                                        short=False),
+                                    MessageAttachmentField(
+                                        title="Users",
+                                        value="27 Million",
+                                        short=True)])])
+        self.bot.send_message(message)
+
+        attachments = """[{"fallback": "fallback", "color": "#36a64f",
+ "author_name": "John Doe", "author_link": "https://example.com/author",
+ "author_icon": "https://example.com/author.jpg", "title": "Title",
+ "title_link": "https://example.com", "text":
+ "Go to Example Website", "image_url": "https://example.com/image.jpg",
+ "thumb_url": "https://example.com/thumb.jpg", "fields": [{"title":
+ "Established", "value": "2008", "short": false},
+ {"title": "Users", "value": "27 Million",
+ "short": true}], "footer": "Footer text", "footer_icon":
+ "https://example.com/footer.jpg"}]""".replace('\n', "")
+
+        self.bot.slack_client.api_call.assert_called_with('chat.postMessage',
+                                                          channel='channel_id',
+                                                          as_user=True,
+                                                          attachments=(
```

Comment:
One more nitpick does attachments have to be within brackets, does json.dumps() return a tuple? In Python brackets create a tuple object, so there is a difference between `"string" and ("string")`. I pretty certain single element tuples are fully compatible with how you're using it but it still feels a bit wrong to 'expect' something that may not being returned

Reply:
This is purely to keep the line under the 80 character limit

Reply:
I don't think you need the brackets, a new line on its own is fine as it is already enclosed with brackets.